x-pack/plugins/security_solution/public/management/pages/event_filters/store/reducer.test.ts

```diff
@@ -50,6 +51,7 @@ describe('reducer', () => {
             ...entry,
             name: nameChanged,
           },
+          newComment,
```

Comment:
Needs a new field for the new comment string as the existing comments has more fields needed to be displayed on AddExceptionComments component